docs/guides/nrfconnect_examples_software_update.md

```diff
@@ -118,7 +118,72 @@ Complete the following steps to perform DFU using mcumgr:
             hash: cbd58fc3821e749d3abfb00b3069f98c078824735f1b2a333e8a1579971e7de1
         Split status: N/A (0)
 
-7.  Reset the device with the following command to let the bootloader swap
+> **_NOTE:_** In case you use nRF5340DK board that supports multi-image device firmare upgrade,
+> complete 7-9 steps. In other case you should omit them and go to the step 10.
+
+7.  Upload the network core firmware image to the device by running the following command in
+    your example directory:
+
+        $ sudo mcumgr --conntype ble --hci ble-hci-number --connstring peer_name='ble-device-name' image upload build/zephyr/net_core_app_update.bin -n 1
+
+    The operation can take few minutes. Wait until the progress bar reaches
+    100%. 
+
+8.  Obtain the list of images present in the device memory by running following
+    command:
+
+        $ sudo mcumgr --conntype ble --hci ble-hci-number --connstring peer_name='ble-device-name' image list
+
+    The displayed output contains the old application image in slot 0 that is currently
+    active, new application image in slot 1 in pending state and the new network image in slot 1, which is not active yet (flags field
```

Comment:
```suggestion
    active, the new application image in slot 1 in pending state, and the new network image which is in slot 1 and not active yet (flags field
```
The previous sentence end is correct if the not active part referred to the slot 1 and not to the network image.